### PR TITLE
feat(ml model): updating view of ml model feature list

### PR DESCRIPTION
**Pull request by gabe-lyons**
Updates the view of the ml model's features to align with feature table list. Refactors the code of [MlFeatureTableFeatures.tsx](https://github.com/datahub-project/datahub/pull/6576/files#diff-5dd7de34f24b1eb3a2772d5feb36c27316cdb95b33e8f698c765093e321b7e58) to factor out the table itself into a reusable component.

CleanShot 2022-11-29 at 16 15 22

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: datahub-web-react/src/app/entity/mlFeatureTable/profile/features/TableOfMlFeatures.tsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+            />
+        ),
+        onCell: onTagTermCell,
+    };
+
+    const termColumn = {
+        width: 125,
+        title: 'Terms',
+        dataIndex: 'glossaryTerms',
+        key: 'glossarTerms',
```
Comment (aditya-radhakrishnan) [inline datahub-web-react/src/app/entity/mlFeatureTable/profile/features/TableOfMlFeatures.tsx:113]: is this misnamed?

Comment (aditya-radhakrishnan) [reply datahub-web-react/src/app/entity/mlFeatureTable/profile/features/TableOfMlFeatures.tsx:113]: Ah I see it's like that in the original as well. Can we change it?

File: datahub-web-react/src/graphql/mlModel.graphql [hunk 2]
```diff
@@ -8,7 +8,21 @@ query getMLModel($urn: String!) {
             ...partialLineageResults
         }
         features: relationships(input: { types: ["Consumes"], direction: OUTGOING, start: 0, count: 100 }) {
-            ...fullRelationshipResults
+            start
```
Comment (aditya-radhakrishnan) [inline datahub-web-react/src/graphql/mlModel.graphql:11]: Nice presumably this reduces latency?

**Review by aditya-radhakrishnan — APPROVED** [review]
LGTM but if we could rename that one field that would be great

File: datahub-web-react/src/app/entity/mlFeatureTable/profile/features/TableOfMlFeatures.tsx (hunk 1 above)
Comment (gabe-lyons) [reply datahub-web-react/src/app/entity/mlFeatureTable/profile/features/TableOfMlFeatures.tsx]: done!

**Pull request merged**